src/openrct2/actions/CheatSetAction.cpp

```diff
@@ -357,12 +365,40 @@ ParametersRange CheatSetAction::GetParameterRange(CheatType cheatType) const
         case CheatType::ForceWeather:
             return { { 0, EnumValue(WeatherType::Count) - 1 }, { 0, 0 } };
         case CheatType::SetForcedParkRating:
-            return { { 0, 999 }, { 0, 0 } };
+            return { { -1, 999 }, { 0, 0 } };
         case CheatType::CreateDucks:
             return { { 0, 100 }, { 0, 0 } };
-        default:
+        case CheatType::RemoveDucks:
+            [[fallthrough]];
+        case CheatType::ClearLoan:
+            [[fallthrough]];
+        case CheatType::RemoveAllGuests:
+            [[fallthrough]];
+        case CheatType::OwnAllLand:
+            [[fallthrough]];
+        case CheatType::WinScenario:
+            [[fallthrough]];
+        case CheatType::HaveFun:
+            [[fallthrough]];
+        case CheatType::FixVandalism:
+            [[fallthrough]];
+        case CheatType::WaterPlants:
+            [[fallthrough]];
+        case CheatType::FixRides:
+            [[fallthrough]];
+        case CheatType::RenewRides:
+            [[fallthrough]];
+        case CheatType::TenMinuteInspections:
+            [[fallthrough]];
+        case CheatType::ResetCrashStatus:
+            [[fallthrough]];
+        case CheatType::RemoveLitter:
             return { { 0, 0 }, { 0, 0 } };
+        default:
+            Guard::Assert(false, "Missing validation parameter for cheat: %d", _cheatType.id);
     }
+    // Unreachable
```

Comment:
I'd add an assert here instead

Reply:
I still would need the return there, otherwise `warning C4715: 'CheatSetAction::GetParameterRange': not all control paths return a value` as Guard::Assert is not marked as noreturn